doc/nrf/app_dev/device_guides/nrf91/nrf91_snippet.rst

```diff
@@ -111,22 +149,61 @@ nRF91 modem traces with RTT backend using snippets
 
 The ``nrf91-modem-trace-rtt`` snippet enables the :kconfig:option:`CONFIG_NRF_MODEM_LIB_TRACE` Kconfig option and chooses the RTT trace backend with the necessary Kconfig options.
 
-To enable modem traces with the RTT backend, use the following command pattern, where *board_target* corresponds to your board target and `<image_name>` to your application image name:
+To enable modem tracing with the RTT backend, add the ``nrf91-modem-trace-rtt`` snippet to the :term:`build configuration`.
+This can be done in one of the following ways:
+
+.. tabs::
+
+   .. group-tab:: west
+
+      To add modem traces with the RTT backend when building an application with west, use the following command pattern, where *board_target* corresponds to your board target and `<image_name>` to your application image name:
+
+      .. parsed-literal::
+         :class: highlight
 
-.. parsed-literal::
-   :class: highlight
+         west build --board *board_target* -- -D<image_name>_SNIPPET="nrf91-modem-trace-rtt"
 
-   west build --board *board_target* -- -D<image_name>_SNIPPET="nrf91-modem-trace-rtt"
+   .. group-tab:: CMake
+
+       To add the modem traces with the RTT backend snippet when building an application with CMake, add the following command to the CMake arguments:
+
+       .. code-block:: console
+
+          -D<image_name>_SNIPPET="nrf91-modem-trace-rtt" [...]
+
+       To build with the |nRFVSC|, specify ``-D<image_name>_SNIPPET="nrf91-modem-trace-rtt" [...]`` in the **Extra CMake arguments** field.
+
+       See :ref:`cmake_options` for more details.
 
 .. _tfm_enable_share_uart:
 
 Shared UART for application and TF-M logging
 ********************************************
 
 If you want to activate TF-M logging while having modem traces enabled, it can be useful to direct the TF-M logs to the UART (**UART0**) used by the application.
```

Comment:
```suggestion
If you want to activate TF-M logging while having modem traces with the UART backend enabled, it can be useful to direct the TF-M logs to the UART (**UART0**) used by the application.
```